### ISSUE TITLE
1-to-Many-Subscriber Notification

### ISSUE THREAD

**Issue by ptrdtznr** [telefonicaid/fiware-orion#3560]
I could not find any information about this scenarios, therefore here we go:

We got 1 publisher of an entity and n (>100) Subscribers to it. Once, the publisher changes the entity, all subscribers will be notified. What happens, if the Orion notifies subscriber 67/100 and the publisher sends a new update?

a) Orion finishes first the 33 other subscribers with the update and than starts with the with the new 100 

b) Orion starts from 1 with the new value? After the "old" notifications has been send we have 2*N messages?

c) Do you have some tests (Jmeter/Postman) to reproduce it?!

**Comment by fgalan** [comment]
I think this is more a question than an issue and should be published in StackOverflow (https://stackoverflow.com). Could you do that, pls? It's basically a copy-paste :). Please, add the "fiware-orion" label to the question.

Once published in SOF, please add a link to it as a comment on this issue. Then, close the issue.

Thanks!

**Comment by ptrdtznr** [comment]
Okay, let me rephrase the questions to an issue.

I am not able to find/ reproduce the above mentioned question with your provided, by the way outdated, Jmeter2 files. They are also not able to open with the latest Jmeter version 5, However I am also not able to find any documentation about the behavior of Orion according to QualityOfServices, which is clearly required from my point of view for a middleware. If you want, I can edit the issue here ;-)

**Comment by fgalan** [comment]
So you have "move" the original post to an issue related to JMeter files in the repository. Is my understanding correct?

Anyway, I think the original post is valuable as Q&A, as illustrates the way Orion works in that case. If you don't want to create a SOF post with it, may I copy-paste to do so (and self-answer)? Of course, it's better always if the original author (you) to do it, but I don't mind to do in this case if you prefer so.

Thanks!
